src/ExceptionBag.php

```diff
@@ -0,0 +1,43 @@
+<?php
+
+namespace WtfPhp\JsonApiErrors;
+
+use Illuminate\Support\Collection;
+use Throwable;
+
+class ExceptionBag implements ExceptionBagInterface
+{
+    protected Collection $bag;
+
+    public function __construct()
+    {
+        $this->bag = new Collection();
+    }
+
+    public function add(Throwable $throwable): void
+    {
+        $this->bag->add($throwable);
+    }
+
+    public function addMultiple(Collection $throwables): void
+    {
+        $throwables->each(function ($throwable) {
+            $this->bag->add($throwable);
+        });
+    }
+
+    public function getAll(): Collection
```

Comment:
We decided to change the return value here to array as well, to be consistent with the addMultiple()